### ISSUE TITLE
Proposal: File Format for SPIFFE Trust Map File

### ISSUE THREAD

**Issue by zeromath** [spiffe/spiffe#228]
Hello! Our team is working with SPIFFE Trust Map. The [SPIFFE bundles](https://github.com/spiffe/spiffe/blob/main/standards/SPIFFE_Trust_Domain_and_Bundle.md#4-spiffe-bundle-format) mentioned that the format of a `<trust_domain_name, bundle>` tuple, but has no indication of how/where to store these tuples.  We find it really convenient to have a standard format for SPIFFE Trust Map File, which will be a good place to keep track of this trust domain-bundle map and has some other benefits. Details about the question and our proposal can be found [here](https://docs.google.com/document/d/1NY1LgS9tzXnXeNyxau8lFjTNyMvHgMJowirhEa4a4Nk/edit?usp=sharing). We'd love to hear about suggestions/thoughts.

**Comment by evan2645** [comment]
Hi @zeromath - are you still interested in pursuing this work? I think it would be good to discuss further if so. I reviewed the doc and it is very clear and easy to parse, thank you for that 🙏

**Comment by zeromath** [comment]
Yeah, definitely. Let me sync with my colleagues, they probably have some updates on this proposal.

**Comment by evan2645** [comment]
Here is a link to the latest proposal doc, which includes suggested text for a new Section 5 in the trust domain and bundle spec: https://docs.google.com/document/d/1QZnUHG9RC6n-I2cvmdZgPCPhmP6PKtMmIVIZ3evi-ac/edit
